openamp/libmetal.cmake

```
@@ -22,9 +22,9 @@
 if(NOT EXISTS ${CMAKE_CURRENT_LIST_DIR}/libmetal)
   FetchContent_Declare(
     libmetal
-    DOWNLOAD_NAME "libmetal-v${OPENAMP_VERSION}.zip"
+    DOWNLOAD_NAME "libmetal-main.zip"
     DOWNLOAD_DIR ${CMAKE_CURRENT_LIST_DIR}
-    URL "https://github.com/OpenAMP/libmetal/archive/v${OPENAMP_VERSION}.zip"
+    URL "https://github.com/OpenAMP/libmetal/archive/"${LIBMETAL_COMMIT}".zip"
```

Will quotes cause download failure? Should we remove the quotes here?
```suggestion
    URL "https://github.com/OpenAMP/libmetal/archive/${LIBMETAL_COMMIT}.zip"
```

---

Done